public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -36,8 +36,9 @@ define(function(require) {
 
     // Update the UI with a "Saving..." indicator whenever we sync a file
     syncManager.on("file-sync-start", function() {
+		$("#navbar-save-indicator").removeClass("hide");
```

Comment:
as this is jQuery, we don't even need two lines here. This should do the trick too:
```
$("#navbar-save-indicator").removeClass("hide").text("{{ fileSavingIndicator }}");
```
However it is puzzling that this ordering matters, and suggests that maybe this fixes a symptom, not the underlying problem. I would caution landing this before we know *why* it works. And once we do know why, this code should get a little code comment to explain the ordering and warn about changing that ordering.